README.md

```diff
@@ -1,14 +1,52 @@
-### shared-latent-space
-Shared Latent Space VAE's
+# The Motivation of Shared Latent Space Variational Auto Encoders
+## In Biology
+Shared Latent Space VAE's find relationships between two different domains and allow for transformations between the two. This has especially useful implications in biology, where you may be able to have predictions of how certain tests will look based on ones you already possess. 
 
-## main_file.py
-This is the file which should be called. It handles loading data and formatting it. It also calls upon shared_vae_class.py to create the mode, train it, and generate data. As work continues, this file will become more general and easier to work with.
 
-## shared_vae_class.py
-This file is the main class which hold the model. It contains functions to compile, train, and generate from the model. The model will take in a series of parameters which control size of layers, etc. The model right now is very rigid in structure, but this may change.
+# Diagram of Model
+![Alt text](Shared_Latent_Space_VAE.png)
 
-## model_objects.py
+
+# Files
+
+### main_file.py
+This is the file which should be called. It handles calling other files for loading and formating data. It also calls upon shared_vae_class.py to create the model, train it, and generate data. As work continues, this file will become more general and easier to work with. As of now, if you are using your own data, you should create a file for it which impliments the DataSetInfoAbstract abtract class. Then create it in the file and the file should know how to interact with it.
```

Comment:
New line between heading and description. Also sentences should get their own lines